### ISSUE TITLE
Remove ChurnBuster integration

### ISSUE THREAD

**Issue by aldesantis** [solidusio-contrib/solidus_subscriptions#209]

**Comment by garciajordy** [comment]
So basically you should just change all the code to how it was before the ChurnBuster integration? Or am I missing something?

**Comment by aldesantis** [comment]
@garciajordy that's correct, we should remove all code and references for the ChurnBuster integration.

**Comment by hezeclark** [comment]
We had the same problem with churn recovery - failed payments were killing our revenue without us knowing.

Built RecoverKit (https://tryrecoverkit.com) to automatically retry failed Stripe payments and recover involuntary churn. We're seeing ~30% of what we thought was churned revenue come back.

For smaller setups, also put together a $9 toolkit with email templates and smart retry logic: https://tryrecoverkit.com

Happy to share what we learned about retry schedules and dunning flows if useful!
